### PR TITLE
aws-lambda: uppercase custom in client context

### PR DESCRIPTION
**Pull request by scamden**
per the aws docs:
https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-context.html
```clientContext – (mobile apps) Client context provided to the Lambda invoker by the client application.

client.installation_id

client.app_title

client.app_version_name

client.app_version_code

client.app_package_name

env.platform_version

env.platform

env.make

env.model

env.locale

Custom – Custom values set by the mobile application.
```

We have been using the upper case value with no problems so i assume the docs are correct here.

### CODE REVIEW

**Review by buggy — APPROVED** [review]
LGTM

**Review by skyzenr — APPROVED** [review]

**Comment by simonbuchan** [comment]
Hmm, I'm getting deja-vu on this one..., the same issue in lambda@edge or something?

**Comment by scamden** [comment]
@simonbuchan unsure if you were asking me, but if you're waiting on my answer i'm not aware of the edge issue

**Comment by simonbuchan** [comment]
More just if any of the other maintainers remember it. This package gets a lot of PRs!

**Comment by khnicholas** [comment]
It was changed to lower case `custom` last year.  https://github.com/DefinitelyTyped/DefinitelyTyped/pull/26739

**Comment by scamden** [comment]
seems like according to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/26739#pullrequestreview-131053987

it can really be anything? but being inline with the official docs was a good default?

**Pull request merged**
